contributor/outline.md

```diff
@@ -1,64 +1,141 @@
+  - Introduce the speaker.
+  - Quick overview of the _Contributor_ role.
+  - Summary of the roles of InnerSource and how each relates to _Contributor_.
+  - Summary of the upcoming segments and what each covers.
 
-## Technical Concerns
-   - Read the CONTRIBUTING.md and follow it.
-    - Keep in mind larger timezone differences when managing your expectations for reaction times.
-   - If you are contributing the fix to an issue, be sure to follow all guidance and instruction documented in the issue.
-   - Update any documentation that goes along with your code.
-   - Take feedback as data that helps you refine your idea/code
-   - Even after merge, if the Host team has any questions or additional help needed from  you for them to understand and maintain what you contributed, be timely and responsive.
-    - the '30 days warranty' pattern appears to be relevant here. Otherwise given the change of teams in person a "fear of maintenance" and the matching "fear to consume iss artifacts" can ensue.
-   - If you are stuck while working on a contribution, then check the project documentation for ways to get support.
-   Don't spin or give up on your valuable addition.
-   - Behave politely in pull request.
-   - Comments - contributions should be documented.
-   - Fixes for typos and builds are great contributions.
-   - Style of code should be similar to what code is already present.
+Outline for this section inspired by the [Trusted Committer introduction video].
 
-   - How acceptable / feasible / desired are upstream connected forks that work in a stable/experimental alternative setup? 
-    - The OSS environment has such forks where the experimental fork accepts more than the stable fork but there is exchange/upstreaming of proven experimental forks. (e.g. Psi/Psi+ or possibly mainline/LTS Linux)
-    - In an Inner Source environment this might be less feasible due to a smaller amount of resources, however might make a pure research / r&d / maintenance transfer easier
+## Becoming an InnerSource Contributor
 
+  - A contributing mindset starts with a shared use mindset.
+    - When you have technical needs that are outside of the domain of your project,
+    look around to see if there is a shared solution that you can use.
+    - Briefly enumerate the benefits of using shared solutions.
+  - Using shared solutions puts you in a position to become a _Contributor_.
+    - There will be times when you wish that the shared solution has some new feature.
+    - Don't _Wait it Out_ or _Work Around_, spend your time and energy making the change directly in the shared project.
+    - Briefly enumerate the benefits of contributing.
+    This should be taken from the **Benefits** segment.
+    - You may need to spend a bit of time explaining these benefits to your management hierarchy in order to have their support to spend your time contributing.
```

Comment:
Nice!  Just do it.  Updated to not make it sound like you're required to get manager permission to contribute.

Reply:
I agree with this, *however*, we need to be cognizant that getting inner source up and running in some orgs is more of a 'getting permissions' than 'being a renegade'.  I like @gruetter's proposed text for that reason.  While *I* like 'Just Do It(tm)', it doesn't mean it's universally applicable to all situations.

Reply:
I think this might depend upon the size of a contribution, i.e. as soon as efforts become larger and take up more time, convincing your management and possibly higher up levels of management becomes more important.
While your immediate line manager may be convinced, the entire thing may become infeasible if incentives are set in a way that would make line managers harm themselves on allowing contributions.

Reply:
Recommend taking any suggestions here to the revised text (present in https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/64).